pkg/provider/azure_vmssflex_cache_test.go

```diff
@@ -103,8 +105,43 @@ var (
 				VMSize: compute.VirtualMachineSizeTypesStandardD2sV3,
 			},
 		},
+		Zones:    nil,
+		Location: to.StringPtr("EastUS"),
+	}
+
+	testVMWithoutInstanceView3 = compute.VirtualMachine{
```

Comment:
Can we wrap these into a function to reduce code redundancy?

Reply:
Sounds good. Just added a helper function to wrap them